sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/tests/WebPubSubEventRequestTests.cs

```diff
@@ -107,8 +107,8 @@ public void TestConnectEventRequestSerialize()
                 {"query2", new string[]{"b1"} },
             };
             var certs = new List<WebPubSubClientCertificate>
-            { 
-                new WebPubSubClientCertificate("111"), 
+            {
+                new WebPubSubClientCertificate("111"),
                 new WebPubSubClientCertificate("222")
```

Comment:
Nit: I recommend trailing commas to make the diff smaller / easier to read / more merge-conflict resilient.

Reply:
Skipping it in the test code for this round of changes.